src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -124,7 +124,7 @@ public async Task ExecuteAsync(string applicationName, string deploymentProjectP
 
             await CreateDeploymentBundle(orchestrator, selectedRecommendation, cloudApplication);
 
-            await orchestrator.DeployRecommendation(cloudApplication, selectedRecommendation);
+            await orchestrator.DeployRecommendation(cloudApplication, selectedRecommendation, deploymentProjectPath);

```

Comment:
Instead of passing `deploymentProjectPath` through all of the code paths I think we can check the `PersistedDeploymentProject` property on the recommendation and if that is true use the CDK project path from the recommendation.

Reply:
Done